diff --git a/node_modules/@vue/babel-preset-app/index.js b/node_modules/@vue/babel-preset-app/index.js
index 23f1d27..299da60 100644
--- a/node_modules/@vue/babel-preset-app/index.js
+++ b/node_modules/@vue/babel-preset-app/index.js
@@ -184,9 +184,9 @@ module.exports = (context, options = {}) => {
   // be accept by webpack / ESLint, and must be processed by Babel first.
   // Chrome 79 is the last Chrome version that doesn't support these syntaxes.
   // So the targets set by the user cannot be higher than Chrome 79.
-  if (!targets.chrome || semver.gt(targets.chrome, '79.0.0')) {
-    targets.chrome = '79.0.0'
-  }
+  // if (!targets.chrome || semver.gt(targets.chrome, '79.0.0')) {
+  //   targets.chrome = '79.0.0'
+  // }
 
   if (process.env.VUE_CLI_BABEL_TARGET_NODE) {
     // running tests in Node.js
